symplyphysics/definitions/wave_equation_in_one_dimension.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (
+    Derivative,
+    Eq,
+    symbols,
+    Function as SymFunction,
+)
+from symplyphysics import (
+    units,
+    Symbol,
+    print_expression,
+)
+
+# Description
+## The wave equation is a second-order linear partial differential equation used to
+## describe the description of waves, including standing wave fields such as mechanical
+## or electromagnetic waves.
+
+# Law: d**2(u(x, t))/dx**2 = (1/v**2) * d**2(u(x, t))/dt**2
+## u(x, t) - factor representing a displacement from rest situation
```

Comment:
`displacement from rest situation` sounds weird. Try to look for some better description of this function.
Also, what's being displaced here?

Reply:
It's the same displacement function as in the [harmonic oscillator equation](https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/harmonic_oscillator_is_second_derivative_equation.py). The $`u`$ function in this equation can be position, pressure, electric or magnetic field. Hence I think it's reasonable to not specify the dimension

Reply:
Yes, indeed.